config/backend-config/multitenant_workspaces_test.go

```diff
@@ -97,11 +97,13 @@ func TestMultiTenantWorkspacesConfig_Get(t *testing.T) {
 	t.Run("ok", func(t *testing.T) {
 		var (
 			secretToken            = "multitenantWorkspaceSecret"
-			workspaceId            = "testWorkspaceId"
+			workspaceID            = "testWorkspaceId"
 			sampleWorkspaceSources = map[string]ConfigT{
-				workspaceId: sampleBackendConfig,
+				workspaceID: sampleBackendConfig,
 			}
+			cpRouterURL = "mockCPRouterURL"
 		)
+		t.Setenv("CP_ROUTER_URL", cpRouterURL)
```

Comment:
Is this needed at all now that we got the URL as a struct attribute?

Reply:
Removed